pkg/ytconfig/generator.go

```diff
@@ -310,7 +312,9 @@ func (g *Generator) GetClusterConnection() ([]byte, error) {
 }
 
 func (g *Generator) GetStrawberryControllerConfig() ([]byte, error) {
-	c := getStrawberryController()
+	var resolver AddressResolver
+	g.fillAddressResolver(&resolver)
+	c := getStrawberryController(&resolver)
```

Comment:
Let's pass not pointer but struct to `getStrawberryController` since we are not changing it inside that function and nil value there is unexpected and will cause panic, so safer and more straightforward is just to just pass `resolver` value here.